src/runtime/c_runtime_api.cc

```diff
@@ -356,6 +356,15 @@ int DGLSetStream(int device_type, int device_id, DGLStreamHandle stream) {
   API_END();
 }
 
+int DGLGetStream(int device_type, int device_id, DGLStreamHandle* stream) {
+  API_BEGIN();
+  DGLContext ctx;
+  ctx.device_type = static_cast<DLDeviceType>(device_type);
+  ctx.device_id = device_id;
+  *stream = DeviceAPIManager::Get(ctx)->GetStream();
```

Comment:
There's no return code.

Also would GetStream() always succeed?  Better to have a sanity check here and return -1 if it fails.

Reply:
`API_END()` includes exception catch and the return code.
`GetStream()` just return a data member in a thread_local which is nullptr in default. so I think it always succeeds.